config/backend/config.yaml

```diff
@@ -14,10 +14,7 @@ metadata:
   # backend.port: '8888'
 
   ## The address the metric endpoint binds to
-  # backend.metrics.address: :8883
-
-  ## If set the metrics endpoint is served securely
-  # backend.metrics.secure: 'true'
+  # backend.metrics.address: :8091
```

Comment:
better change this to `backend.metrics.port` and fix the value in the example as this isn't the address but just the port so we must remove the `:`